### PR TITLE
Replace usage of `activeClassName` in `react-router-dom` v6 ⬆️ 

### PR DESCRIPTION
**Pull request by maoo**

### CODE REVIEW

File: src/ui/views/User/User.jsx [hunk 1]
```diff
@@ -35,9 +35,11 @@ export default function Dashboard(props) {
   const navigate = useNavigate();
 
   useEffect(() => {
-    const id = props.match.params.id;
-    if (id == null) {
+    let id = null;
+    if (!props.match) {
       setIsProfile(true);
+    } else {
+      id = props.match.params.id;
```
Comment (JamieSlome) [inline src/ui/views/User/User.jsx:42]: `props` doesn't exist here anymore. The `useParams()` function is required from `react-router-dom` 👍

**Comment by JamieSlome** [comment]
@maoo - can you take care of the merge conflicts?

File: src/ui/views/User/User.jsx (hunk 1 above)
Comment (maoo) [reply src/ui/views/User/User.jsx:42]: Is this what you mean?

```
  const params = useParams();

  useEffect(() => {
    let id = null;
    if (!params.match) {
      setIsProfile(true);
    } else {
      id = params.match.params.id;
    }
```

Comment (maoo) [reply src/ui/views/User/User.jsx:42]: saw your fix on the reconcile branch. Thank you!

**Comment by maoo** [comment]
> @maoo - can you take care of the merge conflicts?

done!

File: src/ui/components/Sidebar/Sidebar.jsx [hunk 2]
```diff
@@ -39,7 +39,7 @@ export default function Sidebar(props) {
           <NavLink
             to={prop.layout + prop.path}
             className={activePro + classes.item}
-            activeClassName='active'
+            activeclassname='active'
```
Comment (JamieSlome) [inline src/ui/components/Sidebar/Sidebar.jsx:42]: I think the fix here is actually to attach the `exact` property and keep the usage of `activeClassName` the same?

Ref: https://stackoverflow.com/questions/40218050/react-router-link-activeclassname-not-working

Comment (maoo) [reply src/ui/components/Sidebar/Sidebar.jsx:42]: > I think the fix here is actually to attach the `exact` property and keep the usage of `activeClassName` the same?
> 
> Ref: https://stackoverflow.com/questions/40218050/react-router-link-activeclassname-not-working

This post is quite old; my understanding is that `activeClassName` was deprecated, see https://dev.to/gabrlcj/react-router-v6-some-of-the-new-changes-181m#navlink-activeclassname-prop-does-not-exists-anymore

Comment (JamieSlome) [reply src/ui/components/Sidebar/Sidebar.jsx:42]: @maoo - from the article, it looks like we want the following:

```js
className={(navData) => navData.isActive ? "active" : "" }
```

File: src/ui/views/Login/Login.jsx [hunk 3]
```diff
@@ -79,13 +79,13 @@ export default function UserProfile() {
   return (
     <form onSubmit={handleSubmit}>
       <Snackbar
-        open={message}
+        open={true}
```
Comment (JamieSlome) [inline src/ui/views/Login/Login.jsx:82]: This doesn't seem correct? This means the `<Snackbar />` will always appear?

File: src/ui/components/Sidebar/Sidebar.jsx (hunk 2 above)
Comment (JamieSlome) [reply src/ui/components/Sidebar/Sidebar.jsx:42]: > @maoo - from the article, it looks like we want the following:
> 
> ```js
> className={(navData) => navData.isActive ? "active" : "" }
> ```

Just tried, doesn't work...

Comment (JamieSlome) [reply src/ui/components/Sidebar/Sidebar.jsx:42]: @maoo - can we redact this change for now as this doesn't fix the issue and we can look to put into another issue and merge your other changes?

**Comment by JamieSlome** [comment]
@maoo - I've introduced the changes directly to `reconcile` 👍 Thank you ❤️ 

We can now focus on the `activeClassName` issue here...

**Comment by JamieSlome** [comment]
As referenced in the commit message above, `activeClassName` is no longer required and the `<Sidebar />` component is working as expected.

Closing 👍 🎉

**Pull request closed without merging**